public/app/plugins/datasource/postgres/meta_query.ts

```diff
@@ -0,0 +1,176 @@
+export class PostgresMetaQuery {
+  constructor(private target, private queryModel) {}
+
+  getOperators(datatype: string) {
+    switch (datatype) {
+      case 'float4':
```

Comment:
Having metadata returned from API with datatypes like string, numeric could help to get rid of checking specific postgres datatypes here. However, you still need to do it somewhere, but maybe certain parts of this then could be extracted to a common base class.